doc/common/glossary.md

```diff
@@ -116,48 +158,60 @@ Terms not defined by Kubernetes:
 >_*Note:*_ The official [OpenStack Glossary]( https://docs.openstack.org/image-guide/common/glossary.html) is an extensive list of OpenStack-related concepts. Some additional terms used in the Reference Architecture RA-1 or used to relate RA-1 terms with terms defined elsewhere.
 
 <a name="Core"></a>
-**Core (physical)** is an independent computer processing unit that can independently execute CPU instructions and is integrated with other cores on a multiprocessor (chip, integrated circuit die). Please note that the multiprocessor chip is also referred to as a CPU that is placed in a socket of a computer motherboard.
+**Core (physical):** An independent computer processing unit that can independently execute CPU instructions and is integrated with other cores on a multiprocessor (chip, integrated circuit die). Please note that the multiprocessor chip is also referred to as a CPU that is placed in a socket of a computer motherboard.
 
-**Flavor Capabilities** such as (CPU Pinning, NUMA, huge pages): CNTT NFVI Profile
+**Flavor Capabilities** such as (CPU Pinning, NUMA, huge pages): CNTT NFVI Profile.
 
-**Flavor Geometry** Flavor sizing such as number of vCPUs, RAM, disk, etc.
+**Flavor Geometry:** Flavor sizing such as number of vCPUs, RAM, disk, etc.
 
-**[Fluentd](https://www.fluentd.org/)**, a CNCF graduated project, is an open-source data collector for unified logging layer, which allows data collection and consumption for better use and understanding of data.
+**[Fluentd](https://www.fluentd.org/):** An open-source data collector for unified logging layer, which allows data collection and consumption for better use and understanding of data. **Fluentd** is a CNCF graduated project,
 
-**Host profile** is a set of underlay configuration values used to install a new compute host/server. Typically, the host profile configurations match the capabilities of one or more flavors.
+**Host profile:** A set of underlay configuration values used to install a new compute host/server. Typically, the host profile configurations match the capabilities of one or more flavors.
```

Comment:
The definition of "host profile" under "Cloud Infrastructure Hardware Profile" is different. Why define it again?

Reply:
Thanks, we have removed it from here.